workflows/utilities/wf_flye_denovo.wdl

```diff
@@ -123,7 +123,7 @@ workflow flye_denovo {
   # Bandage plot generation
   call task_bandage.bandage_plot as bandage {
```

Comment:
this will fail if flye doesn't generate an assembly. please add a boolean check to see if the assembly was generated before these following steps.